packages/block-library/src/missing/edit.native.js

```diff
@@ -228,7 +246,11 @@ export default compose( [
 		return {
 			isUnsupportedBlockEditorSupported:
 				getSettings( 'capabilities' ).unsupportedBlockEditor === true,
+			isUnsupportedBlockSwitchable:
+				getSettings( 'capabilities' ).unsupportedBlockEditorSwitch ===
```

Comment:
Is there anything we can do to improve the names here while avoiding Jetpack or similiar? I'm not sure "switchable" or "switch" describes this.

Reply:
I really hope so and I'm open to suggestions 😄

This is meant to describe the possibility to activate `Unsupported Block Editor` if it's not active.
(i.e: Jetpack sites can, but self-hosted sites can't)

What about `isUnsupportedBlockEditorActivable`. (Still confusing, "Will it activate the Editor or the feature?")

We could be more descriptive:
`unsupportedBlockEditorCanBeEnabled`.

Reply:
cc @marecar3

Reply:
I think `unsupportedBlockEditorCanBeEnabled` is a definite improvement. 👍  We could maybe tighten that up a bit to `canEnableUnsupportedBlockEditor`?

Another option would be to instead have a single `unsupportedBlockEditorCapability` field that carried an enum-like value across the bridge with values like "supported" "canBeEnabled" "notCapable". I like how this conveys that these are mutually exclusive (at least I think they are, let me know if I'm mistaken about that), whereas the separate boolean fields we have currently (separately representing "supported" and "can-be-enabled") don't make it as clear (based on the types) how they relate to each other. Just a thought, I don't think this is a critical change.

Reply:
> Another option would be to instead have a single unsupportedBlockEditorCapability field that carried an enum-like value across the bridge

This thought came to mind too when I saw this extra info being needed. I don't think it would be a huge refactor, but it touches the other areas with capability dependency, so I decided to no do a refactor of this kind on this PR.

Probably better sooner than later anyway.

> `canEnableUnsupportedBlockEditor`

I like it! Let's settle with this one for now :yay: